tests/test_AERONETTimeSeriesReader.py

```diff
@@ -44,6 +44,7 @@ def test_dl_data_unzipped(self):
                 count += len(ts.data(var))
             self.assertEqual(count, 49965)
             self.assertEqual(len(ts.stations()), 4)
+            self.assertEqual(ts.metadata()["revision"], "220622120000")
```

Comment:
This will fail on a new download. 
```
self.assertGreaterEqual(int(ts.metadata()["revision"]), int("220622120000"))
```
should account for updates.

The same for all similar tests

Reply:
Should be fixed now. I'm unable to merge the PR though for some reason.